app/src/main/java/com/example/playlistmaker/player/data/PlayerInteractorImpl.kt

```diff
@@ -1,11 +1,11 @@
-package com.example.playlistmaker.domain.impl
+package com.example.playlistmaker.player.data
 
-import com.example.playlistmaker.presentation.creators.Creator
-import com.example.playlistmaker.domain.PlayerState
-import com.example.playlistmaker.domain.api.PlayerInteractor
+import com.example.playlistmaker.creator.Creator
+import com.example.playlistmaker.player.domain.PlayerInteractor
+import com.example.playlistmaker.player.domain.PlayerState
 
 class PlayerInteractorImpl : PlayerInteractor {
-    var repository = Creator.providePlayerRepository()
+    private var repository = Creator.providePlayerRepository()
```

Comment:
```suggestion
    private val repository = Creator.providePlayerRepository()
```
Вроде значение нигде не меняется